janus/lib/models/cc_agreement.rb

```diff
@@ -0,0 +1,11 @@
+class CcAgreement < Sequel::Model
+
+  def to_hash
+    {
+      user_email: user_email,
+      cc_text: cc_text,
+      agreed: agreed,
+      project_name: project_name
+    }
```

Comment:
Per https://www.notion.so/ucsfdatascience/Community-Projects-Spec-92aea94eb8944810b375390eac07411d#1dc62853e2394e1ba0ab6dda01ec89b7, could we add an updated_at or created_at (either... just one needed) to this table so that we can determine the most recent entries when setting permissions?